### PR TITLE
feat(statsd.plugin): add Asterisk configuration file with synthetic charts

### PR DESCRIPTION
**Pull request by ilyam8**
##### Summary

ssia

##### Test Plan

The file shared by @ktsaou, he has been using it for a long time and it works with the recent version of Asterisk.

### CODE REVIEW

**Comment by ktsaou** [comment]
Asterisk configuration to enable this:

file `/etc/asterisk/statsd.conf`:

```
general]
enabled = yes                   ; When set to yes, statsd support is enabled
server = 127.0.0.1              ; server[:port] of statsd server to use.
                                ; If not specified, the port is 8125
prefix = asterisk               ; Prefix to prepend to all metrics
;add_newline = no               ; Append a newline to every event. This is
                                ; useful if you want to run a fake statsd
                                ; server using netcat (nc -lu 8125)
```

**Comment by ktsaou** [comment]
Images of it working:

image

image

**Review by vlvkobal — APPROVED** [review]

File: collectors/statsd.plugin/asterisk.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+You only need to configure Asterisk to send statistics to Netdata. The following suffices for `/etc/asterisk/statsd.conf`:
+
+```ini
+[general]
+enabled = yes                   ; When set to yes, statsd support is enabled
+server = 127.0.0.1              ; server[:port] of statsd server to use.
+                                ; If not specified, the port is 8125
+prefix = asterisk               ; Prefix to prepend to all metrics
+```
+
```
Comment (kickoke) [inline collectors/statsd.plugin/asterisk.md:31]: I would like to include a sentence that describes the result. Does the Asterisk application open a server on 127.0.0.1:8125 and Netdata listens on that server? If yes, how does Netdata recognize that?

**Review by kickoke — COMMENTED** [review]
Added more structure in the configuration section among other feedback.

File: collectors/statsd.plugin/asterisk.md (hunk 1 above)
Comment (ilyam8) [reply collectors/statsd.plugin/asterisk.md:31]: Netdata is a StatsD server that listens on 127.0.0.1:8125. Asterisk is a client.

```cmd
ilyam@deb-fra1-01:~$ sudo ss -ntlp | grep netdata
[sudo] password for ilyam:
LISTEN 0      4096       127.0.0.1:8125       0.0.0.0:*    users:(("netdata",pid=171508,fd=28))
LISTEN 0      4096         0.0.0.0:19999      0.0.0.0:*    users:(("netdata",pid=171508,fd=4))
LISTEN 0      4096            [::]:19999         [::]:*    users:(("netdata",pid=171508,fd=5))
```

File: collectors/statsd.plugin/asterisk.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+You only need to configure Asterisk to send statistics to Netdata. The following suffices for `/etc/asterisk/statsd.conf`:
+
+```ini
+[general]
+enabled = yes                   ; When set to yes, statsd support is enabled
+server = 127.0.0.1              ; server[:port] of statsd server to use.
+                                ; If not specified, the port is 8125
+prefix = asterisk               ; Prefix to prepend to all metrics
+```
```
[the remark this answers is not in the document]
Comment (ilyam8) [reply collectors/statsd.plugin/asterisk.md:30]: What do you mean? The rendered version:

Screenshot 2022-03-15 at 18 49 03

File: collectors/statsd.plugin/asterisk.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+- Asterisk [integrated with StatsD](https://www.asterisk.org/integrating-asterisk-with-statsd/).
+
+## Configuration
+
+Netdata ships
+with [asterisk.conf](https://github.com/netdata/netdata/blob/master/collectors/statsd.plugin/asterisk.conf) with
+preconfigured charts.
+
+You only need to configure Asterisk to send statistics to Netdata. The following suffices for `/etc/asterisk/statsd.conf`:
+
```
[the remark this answers is not in the document]
Comment (ilyam8) [reply collectors/statsd.plugin/asterisk.md:23]: >  is stored, for example,

It is not an example, it is literally `/etc/asterisk/statsd.conf`

File: collectors/statsd.plugin/asterisk.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ilyam8) [reply collectors/statsd.plugin/asterisk.md:30]: Oh, I understood what you meant. Not relevant after https://github.com/netdata/netdata/pull/12381/commits/7b93323b45149f53b89c467f77b994fdab4d7d39

**Review by vlvkobal — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]

**Pull request merged**